pkg/rtc/types/interfaces.go

```diff
@@ -317,7 +317,7 @@ type LocalParticipant interface {
 	MigrateState() MigrateState
 	SetMigrateInfo(previousOffer, previousAnswer *webrtc.SessionDescription, mediaTracks []*livekit.TrackPublishedResponse, dataChannels []*livekit.DataChannelInfo)
 
-	UpdateRTT(rtt uint32)
+	UpdateRTT(rtt uint32, isUDP bool)
```

Comment:
:suggestion: Can we make two APIs?
- `UpdateMediaRTT`
- `UpdateSignallingRTT`
That way, it will be clearer to follow I think.

Media can be TCP too. But, internally (`TransportManager`) can have the logic of deciding to act of media RTT only if it is direct UDP connection.

Reply:
If not the above, suggest renaming the second parameter here to something like `isMedia` rather than `isUDP` as media could be over TCP.